### ISSUE TITLE
auto fill missed mandatory param is a nightmare

### ISSUE THREAD

**Issue by tiger55cn** [ShishirPatil/gorilla#450]
It will cause a lot of risk in production.
Is there a way to avoid it?

**Comment by HuanzhiMao** [comment]
Are you talking about the model hallucinating some parameter value when that information is not provided in the prompt?

**Comment by tiger55cn** [comment]
> Are you talking about the model hallucinating some parameter value when that information is not provided in the prompt?

Yes. Even there is no hallucination, model should not guess the value. In production, everything should be accurate, any guess work should be avoided, otherwise the consequence might be serious.
